src/components/ui/herosection/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import HeroHeading from "../heroheading";
+import Button from "../button";
+import { attachClasses } from "../../../utils/index";
```

Comment:
1. This import statement should be after the classes import
2.  No need to add index in ../../../utils/index - only ../../../utils will work fine